src/go/k8s/Makefile

```diff
@@ -78,12 +79,20 @@ docker-build: test
 docker-push:
 	docker push ${IMG}
 
+# Build the docker image
+docker-build-configurator:
```

Comment:
should we also adjust deploy and deploy-kind?

Reply:
I didn't though about that. I will adjust. Thanks!

Reply:
The deploy doesn't need to know about configurator. The deploy-kind is adjusted.